packages/module/patternfly-docs/content/design-guidelines/design-guidelines.md

```diff
@@ -9,6 +9,22 @@ id: User feedback
 source: design-guidelines
 ---
 
-Design guidelines
+Note: User feedback lives in its own package at [@patternfly/react-user-feedback](https://github.com/patternfly/react-user-feedback).
 
-COMING SOON!
+**User feedback** is a modal component that on-premises products can use to collect asynchronous feedback from users. 
```

Comment:
```suggestion
**User feedback** is a modal component that products can use to collect asynchronous feedback from users. 
```

This extension came about when we realized that multiple on-prem products were interested in leveraging the same feedback experience that was originally developed for a SaaS, but it can be used for both. 👍